### PR TITLE
Update codegen and add satori api generated client.

### PR DESCRIPTION
**Pull request by fsufyan**

### CODE REVIEW

File: codegen/main.go [hunk 1]
```diff
@@ -174,13 +175,6 @@ class ApiClient extends RefCounted:
 		_namespace = p_namespace
 		_server_key = p_server_key
 
-	func _refresh_session(p_session : NakamaSession):
```
Comment (lugehorsam) [inline codegen/main.go:177]: Where does this logic live now?

**Comment by lugehorsam** [comment]
Could you generate a new Nakama client to make sure it is unaffected by the codegen changes?

File: addons/com.heroiclabs.nakama/Satori/SatoriAPI.gd [hunk 2]
```diff
@@ -0,0 +1,1324 @@
+### Code generated by codegen/main.go. DO NOT EDIT. ###
+
+extends RefCounted
+class_name SatoriAPI
+
+# Log out a session, invalidate a refresh token, or log out all sessions/refresh tokens for a user.
+class ApiAuthenticateLogoutRequest extends NakamaAsyncResult:
```
[the remark this answers is not in the document]
Comment (fsufyan) [reply addons/com.heroiclabs.nakama/Satori/SatoriAPI.gd:7]: Agreed.

**Comment by fsufyan** [comment]
> Could you generate a new Nakama client to make sure it is unaffected by the codegen changes?

Already did, the only difference is the removal of session auto refresh code.

**Review by lugehorsam — APPROVED** [review]

**Pull request merged**